### PR TITLE
Fixes to Network Explorer for new testnet

### PR DESCRIPTION
**Pull request by mmsinclair**

### CODE REVIEW

File: explorer/.env.prod [hunk 1]
```diff
@@ -0,0 +1,5 @@
+EXPLORER_API_URL=https://sandbox-explorer.nymtech.net/api/v1
+VALIDATOR_API_URL=https://sandbox-validator1.nymtech.net
```
Comment (tommyv1987) [inline explorer/.env.prod]: @mmsinclair I think you need to remove the `validator1.`
Currently the validator API resolves to `https://sandbox-validator.nymtech.net`

File: .github/workflows/network-explorer.yml [hunk 2]
```diff
@@ -22,6 +22,8 @@ jobs:
         node-version: '14'
```
Comment (tommyv1987) [inline .github/workflows/network-explorer.yml:22]: Is there any reason why we're running version 14 as well?

Comment (tommyv1987) [inline .github/workflows/network-explorer.yml:22]: Is there any reason why we're using node v14 on the workflow?

File: explorer/.env.prod (hunk 1 above)
Comment (tommyv1987) [reply explorer/.env.prod]: Network defaults doesn't have this in it's vec of validators-details like milhon did.

File: .github/workflows/network-explorer.yml (hunk 2 above)
Comment (mmsinclair) [reply .github/workflows/network-explorer.yml:22]: No reason, it could be changed to Node 16 some time

File: explorer/.env.prod (hunk 1 above)
Comment (mmsinclair) [reply explorer/.env.prod]: Ok I've changed this to `https://sandbox-validator.nymtech.net`

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**